api.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://my-nc-news-t13l.onrender.com/api",
+});
+
+const getArticles = () => {
+  let url = "/articles";
+  return api.get(url).then(({ data }) => {
+    return data.articles;
```

Comment:
Nice work extracting out this logic into a separate file, but url should not be defined with a let as this is not something we want to really change.